lib/config.coffee

```diff
@@ -74,7 +74,7 @@ exports.generate = (options, params = {}) ->
 		appUpdatePollInterval: params.appUpdatePollInterval or 60000
 
 		listenPort: 48484
-		vpnPort: options.vpnPort or 1723
+		vpnPort: _.parseInt(options.vpnPort) or 1723
```

Comment:
@abresas The only reason is consistency, since all the CLI/SDK modules use `_.parseInt`. The function is just a wrapper around `parseInt(x, 10)`, so I guess it has the little benefit of protecting you from forgetting the base.